docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst

```diff
@@ -0,0 +1,66 @@
+|Google logo|
```

Comment:
All of the file accessor plugin stuff was pulled out of master. Maybe this branch is not completely up to date?

Reply:
When was that done?

Reply:
Getting master into this branch was a mess.

Reply:
Looks like it happened on Sept 4: https://github.com/google/earthenterprise/commit/ec83a3f1d7e68e6fb9b7419e50f73cc1c9cbc514

Reply:
I can try to make another merge from master.

Reply:
Hypothetically you'll eventually merge master into the QT4 branch again at some time in the future, so you could possibly wait until then.

Reply:
Right. The next step was to create a wip toward master. Changes that are unintentional should be fairly obvious, and can be fixed at that point. For now I am going to check if I can pull anything else from master, and then we will fix things as part of the next PR.